### PR TITLE
Update .travis-ci.sh 

### PR DESCRIPTION
**Pull request by abraunegg**
* Update Travis CI build script to use an LDC version string
* Update LDC to use 1.14.0

### CODE REVIEW

**Comment by norbusan** [comment]
Hmm, 1.12.0 was the last were aarch binaries were provided it seems ... Either we switch versions for this aarch, or remain with 1.12.0

**Comment by abraunegg** [comment]
@norbusan
Yep - I was trying to get things to use the latest, based on the LDC site it's 1.1.3 for armhf releases .. non urgent, but I was wanting to keep the readme / build tests using the same versions

File: README.md [hunk 1]
```diff
@@ -134,8 +135,9 @@ sudo apt install libnotify-dev
 ```text
 sudo apt-get install libcurl4-openssl-dev
 sudo apt-get install libsqlite3-dev
-wget https://github.com/ldc-developers/ldc/releases/download/v1.11.0/ldc2-1.11.0-linux-aarch64.tar.xz
-tar -xvf ldc2-1.11.0-linux-aarch64.tar.xz
+sudo apt-get install libxml2
+wget https://github.com/ldc-developers/ldc/releases/download/v1.13.0/ldc2-1.13.0-linux-aarch64.tar.xz
+tar -xvf ldc2-1.13.0-linux-aarch64.tar.xz
```
Comment (norbusan) [inline README.md]: Shouldn't that be 1.14.0 to be consistent with the travis test file? Ir do you want the same version for all in the README file?

File: README.md [hunk 2]
```diff
@@ -206,15 +208,15 @@ sudo make install
...
+make DC=~/ldc2-1.13.0-linux-armhf/bin/ldmd2
 sudo make install
 ```
 
 #### ARM64 Architecture
 ```text
 git clone https://github.com/abraunegg/onedrive.git
 cd onedrive
-make DC=~/ldc2-1.11.0-linux-aarch64/bin/ldmd2
+make DC=~/ldc2-1.13.0-linux-aarch64/bin/ldmd2
```
Comment (norbusan) [inline README.md]: Same here

File: README.md (hunk 1 above)
Comment (abraunegg) [reply README.md]: PR #413 updates things to 1.14.0, so this updates everything to "1.13.0" + the travis ci process to use the 'right versions'
I will be applying this PR, then #413 in that order

**Pull request merged**